src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -128,8 +142,10 @@ the MIT License. See LICENSE in the project root for license information. -->
                                        Margin="10,0,0,0"/>
                         </Grid>
                     </StackPanel>
-                    <TextBox x:Uid="Profile_Padding"
-                             Text="{x:Bind State.Profile.Padding, Mode=TwoWay}"/>
+                    <CheckBox x:Uid="Profile_RetroTerminalEffect"
+                              IsChecked="{x:Bind State.Profile.RetroTerminalEffect, Mode=TwoWay}"/>

```

Comment:
For a follow-up, maybe we should have the retro effect on the advanced page? I'm thinking about the future world where there's the retro effect _or any dang shader you want_, that should go in the advanced settings for sure.

Reply:
Funny story about that. Originally, it was gonna be its own grouping. But this would be the only setting in that grouping haha. But yeah, once more cool settings like that go in, we can move stuff around.